src/common/arrow/arrow_converter.cpp

```diff
@@ -123,6 +123,9 @@ void SetArrowFormat(DuckDBArrowSchemaHolder &root_holder, ArrowSchema &child, co
 	case LogicalTypeId::HUGEINT:
 		child.format = "d:38,0";
 		break;
+	case LogicalTypeId::UHUGEINT:
+		child.format = "d:38,0";
```

Comment:
I wonder if this might be a problem
treating hugeint and uhugeint the same in arrow

Reply:
Looking into this a bit more, I've discovered that treating `UHUGEINT` as a `DECIMAL` type does indeed cause some issues as it treats the value as signed. For `HUGEINT` that's mostly okay, even if there is no conversion back to `HUGEINT` from arrow, because the values are pretty much the same. There doesn't appear to be a really good way to handle this. However, there are two more obvious solutions:
- Set the arrow format for `UHUGEINT` to be the same as `DOUBLE` ("g").
- Remove `UHUGEINT` from the supported Arrow types.

Either way there is no clear way of converting back from Arrow-schema to `UHUGEINT` or `HUGEINT`.

Reply:
I think removing UHUGEINT from the supported Arrow types for now sounds like the best option, throwing a NotImplementedException